tokio/src/sync/mpsc/bounded.rs

```diff
@@ -699,6 +699,34 @@ impl<T> Sender<T> {
     pub fn same_channel(&self, other: &Self) -> bool {
         self.chan.same_channel(&other.chan)
     }
+
+    /// Returns the number of available permits in the channel.
+    ///
+    /// # Examples
```

Comment:
It's fine that you use `reserve` in the example, but it'd be nice to mention up here too.

Reply:
I've tweaked the docs a bit. Let me know what you think.